sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/rules/QueryExecutionMetering.scala

```diff
@@ -108,6 +108,12 @@ case class QueryExecutionMetering() {
   }
 }
 
+object QueryExecutionMetering {
+  private val _queryExecutionMeter = QueryExecutionMetering()
+
+  def queryExecutionMeter: QueryExecutionMetering = _queryExecutionMeter
```

Comment:
Is this in preparation for other subsequent PRs? Will `queryExecutionMeter` be invoked/used elsewhere?

How about defining it directly as 


```
object QueryExecutionMetering {
  val queryExecutionMeter = QueryExecutionMetering()
}
```